lib/node_modules/@stdlib/array/base/join/README.md

```diff
@@ -0,0 +1,122 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# join
+
+> Return a string created by joining array elements using a specified separator.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var join = require( '@stdlib/array/base/join' );
+```
+
+#### join( x, separator )
+
+Returns a string created by joining array elements using a specified separator.
+
+```javascript
+var x = [ 1, 2, 3, 4, 5, 6 ];
+
+var out = join( x, ',' );
+// returns '1,2,3,4,5,6'
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+## Notes
+
+-   If provided an array-like object having a `join` method, the function defers execution to that method and assumes that the method API has the following signature:
+
+    ```text
+    x.join( separator )
+    ```
+
+-   If an array element is either `null` or `undefined`, the function will serialize the element as an empty string.
+
+</section>
+
+<!-- /.notes -->
+
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var join = require( '@stdlib/array/base/join' );
+
+var x = [ 0, 1, 2, 3, 4, 5 ];
```

Comment:
It is project convention that the examples in the `examples/index.js` file match what is inside the code block of the `## Examples` section. Would you mind replacing this code block with the contents of the `examples/index.js` file, please? (note: we do use `var join = require( '@stdlib/array/base/join' );` instead of the relative require path for the package that is used in `examples/index.js`)